frontend/src/routes/login.lazy.tsx

```diff
@@ -65,18 +66,94 @@ function Login() {
             Log in
           </h1>
           <div className="card flex flex-col gap-3">
-            {data?.errors?.root && (
-              <div style={{ color: "red" }}>{data.errors.root}</div>
-            )}
-            <div>
+            <Form.Root>
+              {data?.errors && (
+                <Form.Message
+                  className="text-red-500 text-sm"
+                  match={"valueMissing"}
+                >
+                  {data.errors.root}
+                </Form.Message>
+              )}
+
+              <Form.Field className="my-5" name="username">
+                {!!data?.errors?.username && (
+                  <Form.Message
```

Comment:
You put the error message for the `username` field on top of the field, it should be in the bottom like you did in `password`

Reply:
yes yes